layouts/shortcodes/blocks/feature.html

```diff
@@ -2,10 +2,12 @@
 {{ $url_text := .Get "url_text" -}}
 <div class="col-lg-4 mb-5 mb-lg-0 text-center">
 <div class="mb-4 h1">
+{{ with .Get "url" }}<a class="text-white" target="_blank" href="{{ . }}">{{ end }}
```

Comment:
Same here: without shortcode parameter `url`, we end up with invalid HTML markup.

Reply:
Done